src/eureka/S5_lightcurve_fitting/differentiable_models/StarryModel.py

```diff
@@ -185,8 +195,51 @@ def setup(self):
             planet.theta0 = 180.0
             planet.t0 = temp.t0
 
+            # Pixel sampling setup
+            if 'pixel_ydeg' in self.paramtitles:
+                # Get pixel transform matrix and number of pixels
+                if self.ydeg == 2:
+                    self.oversample = 3
+                else:
+                    self.oversample = 2
```

Comment:
In the else part of this if/else statement, were you assuming ydeg=1 or ydeg>2? I'm trying to figure out how to generalize this to any random ydeg

Reply:
I've updated this to be oversample = 3 always to use the appropriate 4L^2 scaling for the number of pixels for an equiangular grid

Reply:
I've changed this to be "oversample=3" by default but allow it to be changed in the run parameters file, as oversample=3 may be more than needed for l>2